commcare_connect/opportunity/models.py

```diff
@@ -75,8 +75,8 @@ class Opportunity(BaseModel):
     total_budget = models.IntegerField(null=True)
     api_key = models.ForeignKey(HQApiKey, on_delete=models.DO_NOTHING, null=True)
     currency = models.CharField(max_length=3, null=True)
-    auto_approve_visits = models.BooleanField(default=False)
-    auto_approve_payments = models.BooleanField(default=False)
+    auto_approve_visits = models.BooleanField(default=True)
```

Comment:
If we are changing all values to true, and removing all ability to set it to false, why do we need this column still? It is impossible for this value to be false after the PR?

Reply:
Yeah, I deliberated on whether to remove it completely but kept it in case there are ideas to use it again for any specific project (or usecase)